Emby.Server.Implementations/TV/TVSeriesManager.cs

```diff
@@ -156,7 +156,7 @@ public IEnumerable<Episode> GetNextUpEpisodes(NextUpQuery request, User user, IE
                         return i.Item1 != DateTime.MinValue;
                     }
 
-                    if (alwaysEnableFirstEpisode || i.Item1 != DateTime.MinValue)
+                    if (alwaysEnableFirstEpisode || (i.Item1 != DateTime.MinValue && (request.NextUpDateCutoff is null ||  i.Item1.Date > request.NextUpDateCutoff)))
```

Comment:
I also changed it to `>=`. From the docs it looks like it should be inclusive?
```suggestion
                    if (alwaysEnableFirstEpisode || (i.Item1 != DateTime.MinValue && i.Item1.Date >= request.NextUpDateCutoff))
```